0021-hashed-timelock-agreements/0021-hashed-timelock-agreements.md

```diff
@@ -139,10 +139,11 @@ Third-party notaries could be used with simple or more complex payment channels.
 
 ### Third Party Payment Channels
 
-Falling in-between the guarantees of "Third Party Escrow" and "Notarized Payment Channels", a "Third Party Payment Channel" escrows money in a payment channel, but gives a third party the authority to create claims. The third party functions as a notary by deciding whether fulfillments were presented on time, but has the additional responsibility of signing the claims on the channel.
+Falling in-between the guarantees of Third Party Escrow and Notarized Payment Channels, a "Third Party Payment Channel" involves escrowing money in a payment channel and giving a third party authority to create claims. To set up the payment channel, the sender funds a temporary account on the ledger as in a Simple Payment Channel. However, the third party's key is authorized to create claims on the channel balance instead of using the sender's public key.
 
-The receiver doesn't have to trust the sender to give them a claim after they present the fulfillment; the trust is instead placed in the third party. This means that the receiver doesn't have to trust the sender at all, and they only need to trust the third party for in-flight money.
+When the sender prepares a transfer, they send a message to the recipient including the hashlock and timeout. To execute the transfer, the recipient submits the preimage to the third party. If the third party gets the preimage before the timeout, they pay the recipient from the sender's funds by signing a new claim.
```

Comment:
Actually the third party is required to act like a message broker; if the receiver acts on just a message from the sender then they don't know that the sender told the third party about it. The receiver could later submit the fulfillment to the third party, only to hear that it was never even prepared.